modules/apps/batch-engine/batch-engine-service/src/test/java/com/liferay/batch/engine/internal/writer/JSONBatchEngineExportTaskItemWriterImplTest.java

```diff
@@ -1,110 +0,0 @@
-/**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
- * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
- */
-
-package com.liferay.batch.engine.internal.writer;
```

Comment:
I am scared of deleting these tests...

Reply:
I pushed forced with a new approach based on portal.properties. This time the unit tests are untouched.